lib/idris-controller.coffee

```diff
@@ -46,6 +47,9 @@ class IdrisController
       @messages.attach()
       @messages.hide()
 
+  stopCompiler: =>
+    @model?.stop()
```

Comment:
It triggers the message: It looks like the compiler was closed or crashed. What do you think? It could be good to know that it worked. Could be distracting on the other hand though. But having to kill the compiler is distracting anyway probably :D